mesop/bin/bin.py

```diff
@@ -30,13 +30,45 @@
 def main(argv: Sequence[str]):
   if len(argv) < 2:
     print(
-      """\u001b[31mERROR: missing command-line argument to Mesop application.\u001b[0m
+      """\u001b[31mERROR: missing command-line argument to Mesop.\u001b[0m
 
-Example command:
+Example run command:
 $\u001b[35m mesop file.py\u001b[0m"""
     )
     sys.exit(1)
 
+  if argv[1] == "init":
+    if len(argv) > 3:
+      print(
+        f"""\u001b[31mERROR: Too many command-line arguments for mesop init.\u001b[0m
+
+Actual:
+$\u001b[35m mesop {" ".join(argv[1:])}\u001b[0m
+
+Re-run with:
+$\u001b[35m mesop init file.py\u001b[0m"""
+      )
+      sys.exit(1)
+
+    filename = argv[2] if len(argv) == 3 else "main.py"
+    absolute_path = make_path_absolute(filename)
+    if os.path.isfile(absolute_path):
+      print(f"\u001b[31mERROR: {filename} already exists.\u001b[0m")
+      sys.exit(1)
+    with open(
+      os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        "../examples/starter_kit/starter_kit.py",
```

Comment:
Ah interesting. Didn't know we included the examples in the Mesop PIP package.

Reply:
yeah, i forgot we included it too. I think it's done here: https://github.com/google/mesop/blob/b47ea60130b9d082e088e276d317c08ddce08bfc/mesop/pip_package/build_pip_package.sh#L74

we could exclude the other examples, but it's probably not too important to trim our binary size (yet)